sdk/messaging/azservicebus/admin/admin_client.go

```diff
@@ -21,8 +22,12 @@ type Client struct {
 	em atom.EntityManager
 }
 
+type RetryOptions = utils.RetryOptions
```

Comment:
Missing doc comment.  I just copy the one from the aliased type.  It's a shame VSCode doesn't propagate the comment for you if it's missing from the outer type.

Reply:
Added doc comment. I'll remove this in the next PR when I get this HTTP client consistent with the actual azcore-way.